test/utils/mocks/account.ts

```diff
@@ -8,6 +8,14 @@ export const MockAccountUndeployed: Account = {
   type: 'erc7579-implementation',
 }
 
+export const MockKernelAccountUndeployed: Account = {
```

Comment:
we can remove the undeployed class right?

Reply:
Yep